### ISSUE TITLE
sample

### ISSUE THREAD

**Issue by kitesoft** [ashutosh1211/shared_preferences_desktop#1]
hi
thank you for greate job
can you provide a sample or dependencies configuration?
thanks in advance

**Comment by ashutosh1211** [comment]
I have tested this with the testbed application that comes with the source code of flutter-desktop-embedding.

Steps to use this Plugin:

1. Clone the repo:
https://github.com/google/flutter-desktop-embedding

2. Copy the windows fde plugin code to this dir.
../flutter-desktop-embedding/plugins/flutter_plugins/

3. Use this plugin in testbed application:
...
shared_preferences: ^0.5.6+3
  shared_preferences_fde:
    path: ../plugins/flutter_plugins/shared_preferences_fde
...

You are good to go. :)

**Comment by atfa** [comment]
When I build the app:
`flutter build windows`
I got this error when I exec: 
` C:\Users\username\projects\projectname\windows\Flutter\ephemeral\.plugin_symlinks\shared_preferences_fde\windows\shared_preferences_plugin.cpp(254,7): . warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc`

But it is ok when I just run it.
`flutter run -d windows`

**Comment by ashutosh1211** [comment]
Yeah, Please check you have latest code from flutter-desktop-embedding.
Or you can set `<TreatWarningAsError>false</TreatWarningAsError>` in Runner.vcxproj file.

**Comment by atfa** [comment]
> Yeah, Please check you have latest code from flutter-desktop-embedding.
> Or you can set `<TreatWarningAsError>false</TreatWarningAsError>` in Runner.vcxproj file.

Thanks. I modified the file `flutter-desktop-embedding\plugins\file_chooser\windows\plugin.vcxproj` and set `<TreatWarningAsError>false</TreatWarningAsError>`, Compile is ok.
